pylzy/lzy/py_env/py_env_provider.py

```diff
@@ -378,3 +376,51 @@ def _get_module_parents(module: ModuleType) -> Iterable[ModuleType]:
         yield parent
 
         parts.pop()
+
+
+def _check_url_is_local_file(url: str) -> bool:
+    file_scheme = 'file://'
+
+    if not url.startswith(file_scheme):
+        return False
+
+    raw_path = url[len(file_scheme):]
```

Comment:
Why not use `urlparse` here

Reply:
I think it is kinda too excess to use this microscope here.